ui/src/containers/AlertPage.js

```diff
@@ -89,18 +75,27 @@ const AlertStatusIcon = styled.div`
   height: 3rem;
   text-align: center;
 
-  &>span {
+  & > span {
     margin: 0;
   }
 `;
 
-const getAlertStatus = (critical, warning) => (
-  critical > 0
-  ? STATUS_CRITICAL
-  : warning > 0
-  ? STATUS_WARNING
-  : STATUS_HEALTH
-);
+// only compare the id and severity since the rest of the data can change often (like updateAt, description that display time)
+// and we don't want to re-render the whole table every time
+const isEqualAlert = (a = [], b = []) => {
+  if (a.length !== b.length) {
+    return false;
+  }
+  return a.every((alertData) =>
+    b.find(
+      (alert) =>
+        alert.id === alertData.id && alert.severity === alertData.severity,
+    ),
+  );
+};
+
+const getAlertStatus = (critical, warning) =>
+  critical > 0 ? STATUS_CRITICAL : warning > 0 ? STATUS_WARNING : STATUS_HEALTH;
```

Comment:
We have `id` for each alert, maybe we can benefit from it?

Reply:
it's actually the number of alerts, i changed so it's more easy to understand